netann/chan_status_manager_test.go

```diff
@@ -768,6 +768,10 @@ var stateMachineTests = []stateMachineTest{
 			// Finally, assert that enabling the channel doesn't
 			// return an error now that everything is in place.
 			h.assertEnables(newChans, nil, false)
+
+			// The channels should already be enabled, so we don't
+			// expect any more updates.
+			h.assertNoUpdates(h.safeDisableTimeout)
```

Comment:
this one also isnt super clear to me. How does adding an additional assertion fix a flake? ie, if the flake was happening earlier on then im not sure how an additional check at the end helps?